docs/using.rst

```diff
@@ -188,6 +190,24 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant 2 way folder sync
```

Comment:
I think the terminology is a bit off here. There is no such thing as "2 way folder sync" in Vagrant. The thing is call Synced Folder - https://www.vagrantup.com/docs/synced-folders/. _rsync_ is the exception here not the rule and really just a fallback. I think we should not come up with out own terminology here, but stick to Vagrant's terminology.